src/main/java/frc/robot/commands/auto/EngageNoPathplanner.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.auto;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.SimplePresetSteerAngles;
+import frc.robot.commands.BalanceCommand;
+import frc.robot.commands.GoToPose;
+import frc.robot.subsystems.Drivetrain;
+
+public class EngageNoPathplanner extends SequentialCommandGroup{
+
+    private Drivetrain m_drive;
+    private Pose2d rightUnrotatedPose, chargePose, centerPose, rightRotatedPose, leftRotatedPose, leftUnrotatedPose;
+    private Rotation2d rot, startRot;
```

Comment:
These member variables are unneeded. They can just be in the constructor because you don't have any other methods